kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/dto.scala

```diff
@@ -122,6 +122,11 @@ case class BatchRequest(
     conf: Map[String, String],
     args: Seq[String])
 
+case class GetBatchListResponse(
+    from: Int,
+    total: Int,
+    sessions: Seq[Batch])
```

Comment:
after think twice, may be `batches` is better.

We can modify the restful api docs later.

Reply:
I will update it after the doc update.

Reply:
thanks, have updated the docs

Reply:
updated.